web_drop_target/__manifest__.py

```diff
@@ -3,16 +3,20 @@
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 {
     "name": "Drop target support",
-    "version": "10.0.1.0.0",
+    "version": "10.0.1.1.0",
```

Comment:
A question/detail: shoudnot this be 10.0.1.0.1 (do we really need to update the module?)

Reply:
it's an improvement :wink: